addon/components/challenges/add.js

```diff
@@ -0,0 +1,23 @@
+import layout from 'bepstore-goals/templates/components/challenges/add';
+import Ember from 'ember';
+import ClickOutside from '../../mixins/click-outside';
+
+const { computed: { equal } } = Ember;
+
+export default Ember.Component.extend(ClickOutside, {
+  layout,
+
+  isRepo: equal('type', 'Repository'),
```

Comment:
I set the type to 'Repository', cause that was easy to read for the titel of the form